manifests/init.pp

```diff
@@ -174,6 +174,8 @@
 #
 # $dhcp_interface::             DHCP listen interface
 #
+# $dhcp_additional_interfaces:: Additional DHCP listen interfaces (in addition to dhcp_interface). Note: as opposed to dhcp_interface *no* subnet will be provisioned for any of the additional DHCP listen interfaces. Please configure any additional subnets using `dhcp::pool` and related resource types (provided by the theforeman/puppet-dhcp module).
```

Comment:
Could you wrap this so it's not such a long line?

Reply:
Sure, how should I wrap it? At 140 characters, like this?

```
# $dhcp_additional_interfaces:: Additional DHCP listen interfaces (in addition to dhcp_interface). Note: as opposed to dhcp_interface *no*
#                               subnet will be provisioned for any of the additional DHCP listen interfaces. Please configure any additional
#                               subnets using `dhcp::pool` and related resource types (provided by the theforeman/puppet-dhcp module).
```

Reply:
That sounds good.